prow/flagutil/kubernetes_cluster_clients.go

```diff
@@ -52,7 +51,6 @@ type ExperimentalKubernetesOptions struct {
 func (o *ExperimentalKubernetesOptions) AddFlags(fs *flag.FlagSet) {
 	fs.StringVar(&o.buildCluster, "build-cluster", "", "Path to kube.Cluster YAML file. If empty, uses the local cluster. All clusters are used as build clusters. Cannot be combined with --kubeconfig.")
 	fs.StringVar(&o.kubeconfig, "kubeconfig", "", "Path to .kube/config file. If empty, uses the local cluster. All contexts other than the default or whichever is passed to --context are used as build clusters. . Cannot be combined with --build-cluster.")
-	fs.StringVar(&o.infraContext, "context", "", "The name of the kubeconfig context to use for the infrastructure client. If empty and --kubeconfig is not set, uses the local cluster.")
```

Comment:
Why?

Reply:
This is a confusing and unnecessary concept. Why do we suddenly need this when we haven't for the past few years?

Reply:
So it was explicit which to use as the infra cluster, not implicit (not all kubeconfig have a current context set, etc...)

Reply:
This is a new feature -- allowing a component to manage prowjobs in a foreign cluster than than the local one.

It is not clear we should have this, and we should not introduce it via a refactor that exchanging client X to Y nor via a refactor that changes the filetype we use to load client credentials.

Reply:
SGTM. Might not have time to review today but will look soon.